src/util.js

```diff
@@ -116,14 +117,15 @@ exports.deserialize = (data, callback) => {
 /**
  * Get the CID of the DAG-Node.
  *
- * @param {Object} dagNode - Internal representation
+ * @param {Buffer} blob - Output from serialize()
```

Comment:
> "Serialized binary data"?

It's binary data always the serialized one? What about "Encoded CBOR"? I don't have a strong opinion on that, so I can also live with keeping it as it currently is.

Reply:
> "Serialized binary data"?

It's binary data always the serialized one? What about "Encoded CBOR"? I don't have a strong opinion on that, so I can also live with keeping it as it currently is.

*Edit*: I should be clearer. When someone is reading just the JSDocs. Would you know what needs to be passed in?

Reply:
I wanted a description that could be used for all IPLD formats.  

So `CBOR` is out.  

`Encoded` begs the question "What is the encoding". By using `Serialized` we are indirectly naming the function to encode.

`serialize` returns a buffer, which is `binary data`.

Reply:
If you want something generic, then I think the original `Output of serialize()` was best. Though I don't know if a generic comment makes sense. Shouldn't the API docs be about that specific implementation? Most people will use this code through js-ipld, there the comment will be generic. But in case you use one format directly, you want to know about the specifics.